spec/flows/household_flow_spec.rb

```diff
@@ -7,24 +7,36 @@
     fill_in_applicant_screen(partner: "Yes")
     fill_in_forms_until(:outgoings)
     fill_in_outgoings_screen
-    # property screen is skipped
-    confirm_screen("vehicle")
-    fill_in_forms_until(:partner_details)
-    fill_in_partner_details_screen
-    # partner dependants and housing benefits screens are skipped
-    confirm_screen("partner_benefits")
     fill_in_forms_until(:partner_outgoings)
     fill_in_partner_outgoings_screen
-    # partner property screen is skipped
-    # partner vehicle screen is skipped
+    confirm_screen("assets")
```

Comment:
We don't technically need a `confirm_screen(X)` if immediately afterwards we have a `fill_in_X_screen` call, because the latter always includes a call to the former. (I know we were already doing it in a couple of places, but I just spotted that it's a bit redundant)